lib/feed.js

```diff
@@ -212,7 +216,7 @@ function atom_1_0(options) {
 
         var entry = [
             { title:        { _attr: { type: 'html' }, _cdata: entries[i].title }},
-            { id:           entries[i].link },
+            { id:           entries[i].id || entries[i].link },

```

Comment:
If you throw an error above at line 109 saying that feed is mandatory, how could link ever be used as an id?